toptal/picasso
# chore: setup changesets

[FX-2161]

### Description

Introduces changesets into Picasso monorepo. It also automatically appends PR body with warning to not forget about peerDependencies. I have tested it out on testing branch.

### How to test

you can check generated PR from changesets 👉 https://github.com/toptal/picasso/pull/2215

### Screenshots

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-2161]: https://toptal-core.atlassian.net/browse/FX-2161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
